ref/src/yppm_driver.f90

```diff
@@ -34,12 +35,12 @@ program yppm_driver
   integer, external :: print_affinity                ! External subroutine
 
   ! Input configuration variables
-  character(len=64) :: namelist_file = "test_input/yppm_0.0.1.nl"
-  character(len=64) :: input_file, output_file
-  integer           :: nl_unit
+  character(len=128) :: namelist_file = "test_input/yppm_0.0.1.nl"
+  character(len=128) :: input_file, output_file
+  integer           :: nl_unit, interpFactor
```

Comment:
`interpFactor` needs a default value (i.e. 0) in case it is not specified in the namelist file.  Previous code should've provided defaults for `input_file` and `output_file` as well.  No logical default for those, but perhaps initialize them to an empty string?